doc/sphinx-guides/source/developers/tips.rst

```diff
@@ -129,6 +129,9 @@ By default, Glassfish reports analytics information. The administration guide su
 Solr
 ----
 
+.. TODO: This section should be moved into a dedicated guide about Solr for developers. It should be extended with
+         information about the way Solr is used within Dataverse, ideally explaining concepts and links to upstream docs.
```

Comment:
Nice comment. Yes, eventually we should dedicate a page to Solr and indexing in the dev guide. There's a lot of complexity that we could try to explain.